sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/SqlTransform.java

```diff
@@ -151,6 +156,35 @@ public static SqlTransform query(String queryString) {
         .build();
   }
 
+  /** Load all UDF/UDAF from {@link UdfUdafRegister}. */
+  public SqlTransform autoRegisterUdfUdaf() {
+    List<UdfDefinition> newUdfs = new ArrayList<>();
```

Comment:
Instead of copying everything, could you just set a bool, then call `env.autoRegisterUdfUdaf()` in `registerFunctions`?